packages/rpc-methods/src/restricted/invokeSnap.ts

```diff
@@ -30,6 +33,17 @@ import { MethodHooksObject } from '../utils';
 
 export const WALLET_SNAP_PERMISSION_KEY = 'wallet_snap';
 
+// Redeclare installSnaps action type to avoid circular dependencies
+export type InstallSnaps = {
```

Comment:
Could we not declare it in `snaps-utils`? Fine if not.

Reply:
I could not declare easily this type on the `snaps-utils` side because it's mostly using the Controller class types to define it and moving the actions type to utils would mean a lot of maintenance.